index.html

```diff
@@ -1,45 +1,146 @@
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <title>
+      Abdo | Portfolio
+    </title>
+  </head>
+  <body>
+    <ul class="nav-list">
+      <li class="nav-item-logo">
+        <span class="logo">Abdo</span>
+      </li>
+    </ul>
+    <div class="main-container">
+      <div class="head-2-container">
+        <p class="hello-title">
+          I’m Abdo<br>
+          Glad to see you!
+        </p>
+      </div>
+      <div class="description-container">
+        <p class="description-main">
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+      </div>
+      <div class="container-link">
+        <a href="#">LET’S CONNECT</a>
+      </div>
+      <div class="container-list-icons">
+        <ul class="content-list-icons">
+          <li class="icon-linkden">
+            <a href="#"></a>
+          </li>
+          <li class="icon-github">
+            <a href="#"></a>
+          </li>
+          <li class="icon-twitter">
+            <a href="#"></a>
+          </li>
+        </ul>
+      </div>
     </div>
-    <div class="description-container">
-      <p class="description-main">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="grid-container">
+      <div class="container-image-bg">
+        <img src="/img/Snapshoot-Portfolio.png" alt="snapshoot-portfolio">
+        <h2>
+          Abdo
+        </h2>
+      </div>
+      <div class="content-side">
+        <div class="type-job">
+          <ul>
+            <li class="title-job">CANOPY
+            </li>
+            <li class="domain-job">Full-Stack Dev
+            </li>
+            <li class="year">2022
+            </li>
+          </ul>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="languages-container">
+          <ul>
+            <li>
+              <span class="html">html</span>
+            </li>
+            <li>
+              <span class="css">css</span>
+            </li>
+            <li>
+              <span class="js">javaScript</span>
+            </li>
+          </ul>
+        </div><button type="button" class="btn-see-project">See Project</button>
+      </div>
```

Comment:
- [ ] Please add the rest of the cards in the work section, there are four in the Figma template. ✋🏽